app/assets/scripts/views/imagery-search.js

```diff
@@ -93,9 +122,15 @@ var ImagerySearch = React.createClass({
 
     this.maps = [
       this.mainMap,
-      ...getImagerySearchLayers().map((o, i) => this.setupMap(this.refs[`mapThumb${i}`], o))
+      ...getImagerySearchLayers().map(
+        (o, i) => this.setupMap(this.refs[`mapThumb${i}`], o, initialMapView)
+      )
     ];
 
+    this.maps.map(
+      (map) => { map.on('moveend', this.onMapChange); }
```

Comment:
Seems a bit unnecessary to add listeners to all maps. Since they're synced using the main map would probably be enough. Or is there a reason behind the approach.